src/applications/personalization/profile/components/military-information/MilitaryInformation.jsx

```diff
@@ -26,8 +26,8 @@ const NotAVeteranAlert = () => {
 
         <p>
           If you think this is an error, call us at{' '}
-          <va-telephone contact={CONTACTS.HELP_DESK} /> (
-          <va-telephone contact={CONTACTS['711']} tty />
```

Comment:
why did the tty prop get removed from this component? It should be in place so that it is formatted correctly and includes TTY in the aria label.